physcraper/writeinfofiles.py

```diff
@@ -138,7 +138,8 @@ def taxon_sampling(filterblast_obj, downtorank=None):
             if filterblast_obj.config.blast_loc == "remote":
                 spn = filterblast_obj.ids.ncbiid_to_spn[key]
             else:
-                spn = filterblast_obj.ids.ncbi_parser.get_name_from_id(key)
+                spn = filterblast_obj.ids.ncbiid_to_spn[key]
+#                spn = filterblast_obj.ids.ncbi_parser.get_name_from_id(key) #TODO this was throwing a pandas error
```

Comment:
The ncbi_parser.get_name_from_id(key) was throwing a pandas error - and this works and made the test pass, but I want to double check that it will not cause errors elsewhere. I think potentially if 'key' is a higher level taxon it will not be present in ncbiid_to_spn?